tools/Importer/BearerTokenHandler.cs

```diff
@@ -150,5 +178,23 @@ public void Dispose(bool disposing)
 
             _disposed = true;
         }
+
+        private static void PrintClaimsFromJwt(string jwtToken)
```

Comment:
Do we need to print this information aways? Can we have a flag to enable this print or not. Just thinking that we can expose someone's information if that person decides to use this tool.

Reply:
It is very handy for test tool. Yes.